AmazonChimeSDKDemo/AmazonChimeSDKDemo/CaptionsModel.swift

```diff
@@ -48,18 +49,11 @@ class CaptionsModel: NSObject {
                     })
                 }
                 
-                var entities = [String]()
-                
-                alternative.entities?.forEach({ entity in
-                    entity.content.split(separator: " ").forEach { word in
-                        entities.append(String(word))
-                    }
-                })
-                
+                let entities = alternative.entities?.flatMap { $0.content.components(separatedBy: " ")}
                 let caption = Caption(speakerName: speakerName,
                                       isPartial: result.isPartial,
                                       content: alternative.transcript,
-                                      entities: entities,
+                                      entities: entities!,
```

Comment:
try - put an empty array as default when entities is nil, e.g. `entities: entities ?? []`

Reply:
this is already addressed in my last commit.